project-template.md

```diff
@@ -24,7 +24,7 @@ Who is currently planning to work on the project? If a project requires speciali
 
 Projects cannot be started until the following participants have been identified:
 * Every project needs a project lead, who is willing to bottom line the project and address any issues which are not handled by other project members.
-* At least two sponsoring TC or GC members. Sponsors are dedicated to attending meetings, reviewing proposals, and in general being aware of the state of the project and its technical details. Sponsors guide the project through the spec process, keep the tracking issue up to date, and help to ensure that relevant community members provide input at the appropriate times.
+* At least two sponsoring TC or GC members (or semantic convention maintainers in the case of semantic convention SIGs). Sponsors are dedicated to attending meetings, reviewing proposals, and in general being aware of the state of the project and its technical details. Sponsors guide the project through the spec process, keep the tracking issue up to date, and help to ensure that relevant community members provide input at the appropriate times.
```

Comment:
Are "semantic convention maintainers",  maintainers of the Semantic Convention SIG/repo as listed here, https://github.com/open-telemetry/semantic-conventions/, or does this also include maintainers for Semantic Convention Subprojects?

Reply:
I added links to clarify, let me know if that helps, thanks